components/google-cloud/aiplatform/sdk/dependencies.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2021 Google LLC. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Setup configuration of  Google Cloud Components client side libraries."""
+
+def make_required_install_packages():
+  return [
+      "kfp>=1.4.0,<2.0.0",
```

Comment:
JFYI: Some projects put requirements in the `requirements.in` or `requirements.txt` files and just read them in setup.py. Tools like `poetry` can be used to manage those.

Reply:
acknowledged.